docs/docs/restapi.md

```diff
@@ -383,4 +383,30 @@ $ curl \
 
 ## Future Improvements
 
-We are actively working on enhancing and expanding the REST API with new features and capabilities. If you have specific use cases, ideas, or feedback that you would like to share, feel free to participate in the discussion on our [Xen Orchestra forum](https://xcp-ng.org/forum/category/12/xen-orchestra).
\ No newline at end of file
+We are actively working on enhancing and expanding the REST API with new features and capabilities. If you have specific use cases, ideas, or feedback that you would like to share, feel free to participate in the discussion on our [Xen Orchestra forum](https://xcp-ng.org/forum/category/12/xen-orchestra).
+
+We encourage feedback and suggestions to help us make this documentation as useful and complete as possible.
+
+### OpenAPI/Swagger
+
+#### What is Swagger?
+
+[Swagger](https://swagger.io/specification/) allows you to describe the structure of an API so that machines can understand. By parsing the structure of the API, Swagger can automatically generate clear, user-friendly, and interactive API documentation. It also makes integration with tools and automatic client library generation possible.
```

Comment:
I believe it's called _OpenAPI_ now, cf https://en.wikipedia.org/wiki/OpenAPI_Specification

Reply:
- OpenAPI is the specification
- Swagger is the "tool" to implement the spec